SocialCareCaseViewerApi/V1/Infrastructure/WarningNote.cs

```diff
@@ -16,46 +16,49 @@ public class WarningNote : IAuditEntity
         public long PersonId { get; set; }
 
         [Column("start_date")]
-        [Required]
         public DateTime? StartDate { get; set; }
 
         [Column("review_date")]
-        [Required]
         public DateTime? ReviewDate { get; set; }
 
         [Column("end_date")]
         public DateTime? EndDate { get; set; }
 
         [Column("individual_notified")]
-        [Required]
-        public Boolean DisclosedWithIndividual { get; set; }
+        public bool DisclosedWithIndividual { get; set; }
```

Comment:
I know a lot of this was made before you touched this code but it could be really good to do define fields that are not required as nullable, this should help us avoid NullReferenceExceptions in the code.

I am happy for this to be part of a separate PR/Refactor piece as it does feel outside scope of this work, I just wanted to make a note of it here.

Reply:
Happy for this to be a separate PR for refactoring 👍🏾